user-guide/Standard_Apps/Monitoring_Solutions/PLM_Tool/Tutorial_Creating_PLM_activity.md

```diff
@@ -0,0 +1,148 @@
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.2.
+
+<!-- Expected duration: TBD -->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.6 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- A Low-Code Apps license.
+- A user account with all [user-definable apps user permissions](xref:DataMiner_user_permissions#modules--user-definable-apps) enabled
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
+
+- [Step 1: Deploy the Microsoft Platform protocol](#step-1-deploy-the-microsoft-platform-protocol)
+- [Step 2: Create an element using the Microsoft Platform protocol](#step-2-create-an-element-using-the-microsoft-platform-protocol)
+- [Step 3: Deploy the PLM package and configure the PLM element](#step-3-deploy-the-plm-package-and-configure-the-plm-element)
+- [Step 4: Configure a PLM resource in the PLM app](#step-4-configure-a-plm-resource-in-the-plm-app)
+
+## Step 1: Deploy the Microsoft Platform protocol
+
+To create a planned maintenance activity, you first need an element that you can schedule the activity for. For this tutorial, a Microsoft Platform element is used. This will allow you to monitor the server used to host the DataMiner Agent. Before you can create this element, you first need to add the protocol to your DataMiner System.
+
+> [!NOTE]
+> If the Microsoft Platform protocol has already been deployed on your DataMiner System, you can skip this step and continue with [step 2](#step-2-create-an-element-using-the-microsoft-platform-protocol).
+
+1. Look up the [Microsoft Platform connector in the Catalog](https://catalog.dataminer.services/details/connector/251).
+
+1. Click the *Deploy* button to deploy the protocol to your DMA.
+
+   > [!TIP]
+   > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
+
+1. [Open DataMiner Cube](xref:Opening_the_desktop_app) and go to *Apps* > *Protocols & Templates*.
+
+1. Right-click the latest Microsoft Platform protocol version and select *Set as Production*.
+
+   > [!TIP]
+   > See also: [Promoting a protocol version to production version](xref:Promoting_a_protocol_version_to_production_version)
+
+## Step 2: Create an element using the Microsoft Platform protocol
+
+1. In DataMiner Cube, right-click the Surveyor and select *New* > *Element*.
+
+1. Specify the following information:
+
+   - Name: `DMA-Localhost`
+   - Protocol: *Microsoft Platform*
+   - Version: *Production*
+   - IP address/host: `127.0.0.1` (for both connections)
+
+     ![Microsoft Platform element configuration](~/user-guide/images/PLM_Tutorial_1.png)
+
+1. Go to the *views* tab and select the view where you want to add the element.
+
+1. Click *Create*.
+
+1. Verify if the element is polling data correctly:
```

Comment:
@ThomasRich-Skyline, what should users do if they see the element is not polling correctly? If this verification step is needed, we also need to mention what users should do if the verification shows that polling is not OK.